docs/include/conf/events/badge-flair.rst

```diff
@@ -0,0 +1,20 @@
+What started as an unofficial, community-organized activity 4 years ago at the first Prague conference, has since become a traditional part of our conferences.
+To participate in the badge flair contest, just decorate your badge and tweet a photo, and you might win a prize!
```

Comment:
Doesn't this flow better as: `just decorate your badge, tweet a photo, and you might win a prize!` ?